src/bos/operators/power_on.py

```diff
@@ -68,21 +73,16 @@ def _act(self, components):
             set_cfs(components, enabled=False, clear_state=True)
         except Exception as e:
             raise Exception("An error was encountered while setting CFS information: {}".format(e)) from e
-        component_ids = [component['id'] for component in components]
-        try:
-            capmc.power(component_ids, state='on')
-        except Exception as e:
```

Comment:
I don't see any error handling that replaces this.  Do we need to have this somewhere?  I'm not sure why the power on operator was the only one with this particular error handling in the first place.

Reply:
All of the error handling is in _power_components in the new power_operator_base. This was part of the problem I was solving. Throwing an exception like this in _act will causes us to live-lock. The error condition is never corrected. However, this was mainly a problem in the status operator.